### PR TITLE
update styles + add notification

### PR DESCRIPTION
**Pull request by taylordolan**
- [x] @tosih 
- [x] @runnabro

https://runnable.atlassian.net/browse/SAN-5345

### CODE REVIEW

File: app/views/todos/show.html.haml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-.container
```
Comment (tosih) [inline app/views/todos/show.html.haml:1]: I don't think we should remove this. You will get an app error if you got to <navi_url>:3000/todos/#

**Review by tosih — COMMENTED** [review]
Also maybe we should remove the stylesheet link to bulma.css

File: app/views/todos/show.html.haml (hunk 1 above)
Comment (taylordolan) [reply app/views/todos/show.html.haml:1]: So I should just put the file back in?

Comment (taylordolan) [reply app/views/todos/show.html.haml:1]: I don't get that issue when I go to that address.

Comment (tosih) [reply app/views/todos/show.html.haml:1]: I mean a number instead of #.

Comment (tosih) [reply app/views/todos/show.html.haml:1]: http://1903l4-rails-starter-staging-runnabledemo.runnableapp.com:3000/todos/1

Comment (tosih) [reply app/views/todos/show.html.haml:1]: If you can put the file back in with simple styling. People can't hit the link via UI, but people familiar with rails will try and hit the direct route if they are curious.

**Pull request merged**